documentation/ipi-install/modules/ipi-install-configuring-the-install-config-file.adoc

```diff
@@ -57,7 +57,7 @@ for i in $(sudo virsh list | tail -n +3 | grep bootstrap | awk {'print $2'});
 do
   sudo virsh destroy $i;
   sudo virsh undefine $i;
-  sudo virsh vol-delete $i --pool default;
+  sudo virsh vol-delete $i --pool $i;
```

Comment:
Unsure about this, as the volume is in $i but the pool should be default right?

Reply:
Your right this should say default as $i is only capturing the volume

Reply:
Applied conditional to >= 4.6 and to < 4.6